data/Makefile

```diff
@@ -6,8 +6,11 @@ ensembl_biomart_geneids_grch37.p13.transcript_info.txt: ensembl_biomart_geneids_
 ensembl_biomart_transcripts.txt: ensembl_biomart_geneids_grch37.p13.transcript_info.txt
 	csvcut -tc transcript_stable_id,gene_stable_id,protein_stable_id,protein_length $< | csvsort -c transcript_stable_id,gene_stable_id,protein_stable_id,protein_length | uniq | csvformat -T > $@
 
-ensembl_biomart_transcripts.json: ensembl_biomart_transcripts.txt ensembl_biomart_pfam_grch37.p13.txt
-	cd ../scripts && python add_nested_domain_to_ensembl_transcript.py > ../data/$@
+ensembl_exon_info.txt: Homo_sapiens.GRCh37.gff3.gz
```

Comment:
is `Homo_sapiens.GRCh37.gff3.gz` $1 and can therefore be passed as input to the .py script?

Reply:
Yes, I have implemented this now in the new commits.